### ISSUE TITLE
final grade

### ISSUE THREAD

**Issue by Tdrinker** [ahemedbullo/CS506-Final-Project#4]
* build, make, test: 10 / 10 - great job! You have significantly improved the documentation for this portion since the midterm report!
* description of data processing and feature engineering done: 10/10
* visualization of data, modeling analysis: 13/15 - I find the feature analysis regarding stock influence very interesting. You can provide more discussion on this result. Image 1 and 2 lacks in depth reflections and discussions.
* modeling results, clear argument proving you achieved your results: 12/15 - great job on the model analysis and performance report. I appreciate the inclusion of baseline. however, i would want to justifications for model choice, analysis for how hyperparameters like learning rate affect your model. Explanations for the observed performances for linear regression versus random forest (whats the nature of these models and why they are related to your task).

45/50
